azDevOps/azure/templates/steps/deploy-terraform-apply.yml

```diff
@@ -0,0 +1,49 @@
+parameters:
+  environment: ''
+  terraform_artefact_path: ''
+  # Parameters specific to terraform as a map
+  tf_job_input_params: ''
+  tf_job_output_params: ''
+
+steps:
+  - bash: pwd; ls -la ; sudo chmod -R 777 ./ ; sudo ls -la; env
```

Comment:
As mentioned on a previous commit, is there a point to this? Doesn't seem a good idea to do this